src/ralph/settings/base.py

```diff
@@ -452,10 +452,16 @@ def get_sentinels(sentinels_string):
     'TRANSITION_ID': os.environ.get(
         'ACCEPT_ASSETS_FOR_CURRENT_USER_TRANSITION_ID', None
     ),
+    'TRANSITION_SIM_ID': os.environ.get(
+        'ACCEPT_SIMCARD_FOR_CURRENT_USER_CONFIG', None
+    ),
     # in_progress by default
     'BACK_OFFICE_ACCEPT_STATUS': os.environ.get(
         'ACCEPT_ASSETS_FOR_CURRENT_USER_BACK_OFFICE_ACCEPT_STATUS', 2
     ),
+    'SIMCARD_ACCEPT_STATUS': os.environ.get(
```

Comment:
`2` stands for `in progress`?

Reply:
Yes, this is this status
https://github.com/allegro/ralph/blob/ng/src/ralph/sim_cards/models.py#L47